src/arco_era5/pangeo.py

```diff
@@ -88,7 +88,7 @@ def run(make_path: t.Callable[..., str], date_range: t.List[datetime.datetime],
 
     recipe = XarrayZarrRecipe(pattern,
                               storage_config=storage_config,
-                              target_chunks={'time': 1},
```

Comment:
More abstraction is needed. While this will work for the model-level dataset, it won't for the single-level.

Reply:
More abstraction was added.